### DISCUSSION TITLE
Create a database backup automatically

### DISCUSSION THREAD

**Discussion by w15eacre** [keepassxreboot/keepassxc#11587] [Ideas]
I couldn't find a feature to automate database backups. I understand that a script could handle it, but it isn't convenient enough.

**Comment by droidmonkey** [comment]
Application Settings, under file management, backup location 

https://keepassxc.org/docs/KeePassXC_UserGuide#_backing_up_a_database_file

Although we ultimately just recommend using a versioned file system like cloud storage or NAS with backups.

**Reply by w15eacre** [reply]
Thanks, I found this option
